templates/web/bexley/waste/container_removal_intro.html

```diff
@@ -0,0 +1,2 @@
+<p>We are unable to collect kitchen caddies. Please dispose of unwanted kitchen caddies by taking them to a Reuse and Recycling Centre or by placing them in your household waste bin</p>
```

Comment:
I think you need full stops at the very end of both of these \<p> blocks.
Otherwise, looks good, and thanks for picking this up!